.gitignore

```diff
@@ -25,7 +25,6 @@ lib/
 node_modules
 release/main.js
 .merlin
-yarn.lock
```

Comment:
I find this a hard thing and will always be subject to preference, so I'll propose that we keep `npm` for now.

Reply:
Alright, this is addressed in 
https://github.com/tcoopman/bucklescript-tea-starter-kit/pull/3/commits/5122b40bc08a0216d183136f66dbc7a0cf5a52d5